modal.js

```diff
@@ -0,0 +1,244 @@
+        <li class="card-detail option">
+          <img src="images/bullets.png" alt="bullet" class="circle" />
+          2015
+        </li>
+      </ul>
+  
+      <p>
+      ${item.description}
+      </p>
+      <div class="language1-inline">
+        <ul>
+          <li class="langu">${item.technologies[0]}</li>
+          <li class="langu">${item.technologies[1]}</li>
+          <li class="langu">${item.technologies[2]}</li>
+        </ul>
+      </div>
+      <button type="button" class="btn-two">See project</button>
+    </div>
+  </div>
+  </div>`;
+  });
+}
+createCards();
+
+const containerMobile = document.querySelector('#projects-section-mobile');
+
+function createCardsMobile() {
+  portfolioDetails.forEach((items) => {
+    containerMobile.innerHTML += `<div class="left1">
+  <div class="project-card">
+    <div class="snap1"><img src="${items.mobileImg}"></div>
+  </div>
+  <div class="left-block">
+    <h2 class="head">${items.name}</h2>
+    <div class="card-detail-box">
+      <ul>
+        <li class="card-detail canopy">${items.Tags[0]}</li>
+        <li class="card-detail option">
+          <img src="images/bullets.png" alt="bullet" class="circle" />
+          ${items.Tags[0]}
+        </li>
+        <li class="card-detail option">
+          <img src="images/bullets.png" alt="bullet" class="circle" />
+          2015
+        </li>
+      </ul>
+  
+      <p>
+      ${items.description}
+      </p>
+      <div class="language1-inline">
+        <ul>
+          <li class="langu">${items.technologies[0]}</li>
+          <li class="langu">${items.technologies[1]}</li>
+          <li class="langu">${items.technologies[2]}</li>
+        </ul>
+      </div>
+      <button type="button" class="btn-one1">See project</button>
+    </div>
+  </div>
+  </div>`;
+  });
+}
+createCardsMobile();
+
+const seeMe1 = document.querySelectorAll('.btn-one1');
+const seeMe2 = document.querySelectorAll('.btn-two');
+
+function createPop(idx) {
+  pop.innerHTML = `
+<div class="container-popup1">
+    <div class="popup-row">
+    <div class="popup-header">
+      <h3 class="popup-header mobile-title">${portfolioDetails[idx].name}</h3>
+      <div class="close-popup">
+      <h2 id="close-btn1">X</h2>
+      </div>
+    </div>
+      <div class="sub-row">
+        <span class="work-sub">${portfolioDetails[idx].Tags[0]}</span>
+        <span class="work-skill">${portfolioDetails[idx].Tags[0]}</span>
+        <span class="work-experience">${portfolioDetails[idx].Tags[2]}</span>
+      </div>
+      <img src="${portfolioDetails[idx].img}" class="work-img popupimg" alt="" />
+      <div class="row-details">
+        <p>${portfolioDetails[idx].moreDetails} </p>
+      </div>
+      <div class="sub-row">
+        <a href="#" class="skills">${portfolioDetails[idx].technologies[1]}</a>
+        <a href="#" class="skills">${portfolioDetails[idx].technologies[1]}</a>
+        <a href="#" class="skills">${portfolioDetails[idx].technologies[1]}</a>
```

Comment:
At this moment you are displaying only 1 technology 3 times:

![image](https://user-images.githubusercontent.com/83596135/180834947-4817c991-ed62-4059-89f2-261b7d0ca63b.png)

- [ ] Please take care at this aspect when refactoring the code and display all the technologies in the list;